ITE/1/README.adoc

```diff
@@ -0,0 +1,925 @@
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="2"]
+|===
+| ITE
+| 1
+
+| Title
+| in-toto Enhancement format
+
+| Sponsor
+| link:https://github.com/santiagotorres[Santiago Torres-Arias]
+
+| Status
+| Active :smile:
+
+| Type
+| Process
+
+| Created
+| 2018-10-04
+
+|===
+
+
+[[abstract]]
+== Abstract
+
+An in-toto Enhancement (ITE) is a design document that describes a new feature
+or aspect of the in-toto framework, a setup of its supply chain or an,
+organizational aspect of the in-toto project.  This ITE describes the structure
+of ITE documents and details the process by which ITEs are created, submitted,
+reviewed, finalized, and maintained.
+
+[[specification]]
+== Specification
+
+=== What is an ITE?
+
+ITE stands for in-toto Enhancement. An ITE is a design document that describes
+a new feature or aspect of the in-toto framework, the in-toto supply chain, or
+the process within in-toto project. An ITE provides a concise technical
+specification of the feature, describes the motivation for the change, and
+discusses the rationale behind the design.
+
+ITEs are the primary mechanism for proposing major new features, for collecting
+community input on an issue, and for documenting the design decisions that have
+gone into in-toto. Each ITE must have at least one <<Sponsor>>. The ITE Sponsor
+is responsible for the ITE overall - building consensus for that ITE within the
+community, documenting dissenting opinions, coordinating contributors work, and
+generally ensuring the ITE meets the style, format, and quality guidelines
+described below. An ITE may also have any number of <<Contributor,
+Contributors>> who help write, implement, discuss, or offer feedback about
+the ITE. One contributor might do only one or any combination of these things
+during any part of the life of a ITE.
+
+Because the ITEs are maintained as text files in a versioned
+repository, their revision history is the historical record of the
+feature proposal footnoteref:[repo, https://github.com/in-toto/ITE].
+
+[NOTE]
+====
+For non-text files added to the repository, such as images, the files must be
+readable with free or open source tools, and should ideally be editable by
+other contributors. For example, use SVG rather than PSD for vector graphics.
+====
+
+=== ITE Types
+
+There are three kinds of ITE:
+
+. A **Standards Track** ITE describes a new feature or implementation
+ for the in-toto specification. It may also describe an interoperability or
+ backwards-compatibility standard which will be supported for the feature in
+ current versions of in-toto, moving forward.
+. An **Informational** ITE describes an in-totodesign issue, or
+ provides general guidelines or information to the in-toto community,
+ but does not propose a new feature. Informational ITEs do not
+ necessarily represent a in-toto community consensus or
```

Comment:
change "a in-toto" to "an in-toto"?
Change this throughout this document for consistency.